### ISSUE TITLE
EXC_BAD_ACCESS

### ISSUE THREAD

**Issue by slynglen** [Alamofire/Alamofire#881]
Whenever I run my project on my device I receive the error "EXC_BAD_ACCESS" - shown in the screenshot. Seems like it works just fine on the simulator.

Im running Xcode 7.1 with Swift 2.1, iOS 9.1 and Alamofire 3.1.0 

![skaermbillede 2015-10-24 kl 01 06 58](https://cloud.githubusercontent.com/assets/3939917/10706888/a3f705a4-79eb-11e5-8b9d-a71bf488994c.png)

Any help is appreciated :-)

**Comment by slynglen** [comment]
I forgot to mention that im making a GET request which make it crash. I really need a solution as I was about to release my app, and Now im stuck.

**Comment by AndrewBarba** [comment]
Can you post the code that builds the request?

**Comment by cnoon** [comment]
Could you try running the `iOS Example` project in the Alamofire workspace under the same conditions to see if you run into the same problem? I just ran the example with the exact setup that you mentioned and everything works exactly as expected. I'm curious if you see the same crash in that example app.

Additionally, as @AndrewBarba mentioned, any source code or sample project you could post would be VERY helpful.

**Comment by slynglen** [comment]
Hi again. 

The iOS Example projects works fine. :-)

Due to privacy I've changed the API endpoint, but it still crashes the same place.

From the applicationDidBecomeActive method I call my singleton class "API" which make the following Alamofire GET request: Alamofire.request(.GET, "http://swifty.dk/test/api") and then it crash like I wrote earlier. The weird thing is that it works in the simulator. :s

I noticed that I got two warnings regarding Alamofire.
1. "PATH.../Pods/Local%20Podspecs/Alamofire.podspec.json: warning: Missing file: PATH.../Pods/Local Podspecs/Alamofire.podspec.json is missing from working copy"

2. "PATH.../Pods/Target%20Support%20Files/Alamofire/Alamofire-Private.xcconfig: warning: Missing file: PATH.../Pods/Target Support Files/Alamofire/Alamofire-Private.xcconfig is missing from working copy"
